### ISSUE TITLE
Hyperparameters provided in the config donot seem to match those provided by the implementation details in the paper

### ISSUE THREAD

**Issue by yxchng** [zhenyuw16/Uni3DETR#7]
For example, the config trains scannet_large for 40 epochs, but the paper says the model is trained for 240 epochs. 

Can the hyperparameters provided in the config be used to reproduce the results in the paper?

Or do I have to modify them based on the paper? But if that is the case, details like what weights to use for each component of the loss are also missing.
